### ISSUE TITLE
ValueError: Cannot feed value of shape (1, 6000, 6000, 3, 1) for Tensor 'annotation:0', which has shape '(?, 6000, 6000, 1)'

### ISSUE THREAD

**Issue by Yaredoh** [shekkizh/FCN.tensorflow#27]
Thanks, for sharing your resource Mr.  Shekkizh. In annotation it's it brings to me error  .. I used python 3.6 and Tensorflow 1.0 . Is it from my working environment ? I only changed the dataset. 

setting up vgg initialized conv layers ...
Setting up summary op...
Setting up image reader...

Found pickle file!
40
8
Setting up dataset reader

Initializing Batch Dataset Reader...
{'resize': True, 'resize_size': 6000}
(40, 6000, 6000, 3)
(40, 6000, 6000, 3, 1)

Initializing Batch Dataset Reader...
{'resize': True, 'resize_size': 6000}
(8, 6000, 6000, 3)
(8, 6000, 6000, 3, 1)
Setting up Saver...

 Traceback (most recent call last):

  File "<ipython-input-1-12b2e3f5c7e5>", line 1, in <module>
    runfile('C:/Users/Yared/Desktop/Project/FCN.py', wdir='C:/Users/Yared/Desktop/Project')

  File "C:\Users\Yared\Anaconda3\lib\site-packages\spyder\utils\site\sitecustomize.py", line 866, in runfile
    execfile(filename, namespace)

  File "C:\Users\Yared\Anaconda3\lib\site-packages\spyder\utils\site\sitecustomize.py", line 102, in execfile
    exec(compile(f.read(), filename, 'exec'), namespace)

  File "C:/Users/Yared/Desktop/Project/FCN.py", line 225, in <module>
    tf.app.run()

  File "C:\Users\Yared\Anaconda3\lib\site-packages\tensorflow\python\platform\app.py", line 44, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))

  File "C:/Users/Yared/Desktop/Project/FCN.py", line 196, in main
    sess.run(train_op, feed_dict=feed_dict)

  File "C:\Users\Yared\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 767, in run
    run_metadata_ptr)

  File "C:\Users\Yared\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 944, in _run
    % (np_val.shape, subfeed_t.name, str(subfeed_t.get_shape())))

ValueError: Cannot feed value of shape (1, 6000, 6000, 3, 1) for Tensor 'annotation:0', which has shape '(?, 6000, 6000, 1)'

**Comment by shekkizh** [comment]
Hi the issue seems related to how your feeding annotation images. The code uses annotations that are gray level images. If possible try converting your annotation to grayscale images.

**Comment by Yaredoh** [comment]
Thank you. That was the problem and I corrected it. 


Now, may be I have a memory problem. I was working on CPU.

setting up vgg initialized conv layers ...
Setting up summary op...
Setting up image reader...
Found pickle file!
2
1
Setting up dataset reader
Initializing Batch Dataset Reader...
{'resize': False, 'resize_size': 6000}
(2, 6000, 6000, 3)
(2, 6000, 6000, 1)
Initializing Batch Dataset Reader...
{'resize': False, 'resize_size': 6000}
(1, 6000, 6000, 3)
(1, 6000, 6000, 1)
Setting up Saver...
**terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Aborted (core dumped)**
